### DISCUSSION TITLE
failed to deploy my project

### DISCUSSION THREAD

**Discussion by kingdavidlevi** [vercel/community#5325] [Help] [answered]
### Summary

Module not found: Error: Can't resolve './Style.css' in '/vercel/path0/src'

### Example

https://e-commerce-app-smoky.vercel.app/

### Steps to Reproduce

<html>
<body>

Failed to compile.
06:27:09.240 |  
06:27:09.240 | Module not found: Error: Can't resolve './Style.css' in '/vercel/path0/src'
06:27:09.241 |  
06:27:09.241 |  
06:27:09.264 | Error: Command "npm run build" exited with 1
06:27:09.601

<br class="Apple-interchange-newline">
</body>
</html>

**Comment by swarnava** [comment, marked as answer]
Hi there,

Can you ensure the filename is correct? i.e it's indeed `Style.css` not `style.css` ?
